sentry-samples/sentry-samples-spring-boot/src/main/resources/application.properties

```diff
@@ -8,5 +8,9 @@ sentry.logging.minimum-event-level=info
 sentry.logging.minimum-breadcrumb-level=debug
 # Performance configuration
 sentry.traces-sample-rate=1.0
+# Database configuration
+spring.datasource.url=jdbc:hsqldb:mem:testdb
+spring.datasource.username=sa
+spring.datasource.password=
```

Comment:
lets add the instruction for setting up hsqldb to README

Reply:
HSQLDB is an in memory database, there is nothing to set up.